### DISCUSSION TITLE
274

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#804] [Comments]
http://jojoldu.tistory.com/274

**Comment by jojoldu** [comment]
작성자: 꿀물

역시 빠르고 생생한 후기 잘 봤습니다.
매번 동욱님 블로그 통해서 도움 많이 받고 있었는데 어제는 직접 만나게 되서 반가웠습니다.
좀 더 얘기를 나누고 싶었는데 아쉽더라구요. 나중에 연락드리고 잠실로 함 찾아갈께요.^^

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

넵^^ 연락주세요! 저도 어제 얘기가 중간에 끝나서 아쉬웠습니다 ㅎㅎ

**Comment by alittlenation** [comment]
좋은정보 감사합니닷
